pkg/controllers/user/alert/initdata.go

```diff
@@ -50,7 +50,7 @@ func initClusterPreCanAlerts(clusterAlertGroups v3.ClusterAlertGroupInterface, c
 			},
 			MetricRule: &v3.MetricRule{
 				Description:    "Etcd member has no leader",
-				Expression:     `etcd_server_has_leader{job="exporter-kube-etcd-cluster-monitoring"}`,
```

Comment:
Is there any possible that we add the right job for this metrics instead of removing the job name?

Reply:
It should only have one etcd job, and the chart may change in the feature, so I delete the condition